conda/recipes/libcuspatial/build.sh

```diff
@@ -1,4 +1,4 @@
 # Copyright (c) 2018-2022, NVIDIA CORPORATION.
 
 # build cuspatial with verbose output
-./build.sh -v libcuspatial tests --allgpuarch -n
+./build.sh -v libcuspatial tests benchmarks --allgpuarch -n
```

Comment:
Does this change alter the contents of the conda package, or are the benchmarks building as a “side effect”?

Reply:
I don't think we need to add them as part of this PR. If there's a need for them in the packages in the future we can revisit the decision. The main goal was to make sure we are compiling all benchmarks in CI so that we don't end up with broken benchmark builds.

Reply:
@vyasr, the `libcudf-tests` package are used in the GPU testing stage of CI.

i.e. here - https://github.com/rapidsai/cudf/blob/edc5062bdcc3e12755603b0ad07a4d271fe95261/ci/gpu/build.sh#L168

then we run the GTests like this: https://github.com/rapidsai/cudf/blob/edc5062bdcc3e12755603b0ad07a4d271fe95261/ci/gpu/build.sh#L183-L187

Reply:
OK got it thanks. Since we don't run benchmarks in CI, I think it's fine to not include the benchmarks in there for now. I suspect that `libcudf` mostly just includes them incidentally because they're part of the same CMake component, not for any particular design reason.